lib/kafka-shared/Makefile

```
@@ -0,0 +1,40 @@
+# cert files (in directory created by `certs` target)
+KAFKA_CA_CERT := $(shell pwd)/certs-0/ca.crt
+KAFKA_CLIENT_KEY := $(shell pwd)/certs-0/tls.key
+KAFKA_CLIENT_CERT := $(shell pwd)/certs-0/tls.crt
+
+CONTEXT := $(shell basename $(shell dirname ${PWD}))
+ENV := $(shell echo ${CONTEXT} | cut -d"-" -f1)
+PROVIDER := $(shell echo ${CONTEXT} | rev | cut -d"-" -f1 | rev)
+
+certs:
+	# obtain the certificates from the kafka pod
+	kubectl exec --context "${CONTEXT}" -n "pubsub" "kafka-shared-0" -- tar cfh - "certs-0/ca.crt" "certs-0/tls.key" "certs-0/tls.crt" | tar xf -
```

use `--container` to avoid output noise: `Defaulted container "kafka" out of: kafka, jmx-exporter` + longopts and splitting up long line

```suggestion
	kubectl exec \
		--context "${CONTEXT}" \
		--namespace "pubsub" \
		--container kafka \
		"kafka-shared-0" \
		-- tar cfh - "certs-0/ca.crt" "certs-0/tls.key" "certs-0/tls.crt" | tar xf -

```

---

about longopts- do you think that developers will benefit from that? That the time we will invest into those kind of enhancements will result in the value for the people or the company? 
[Kubernetes maintainers don't care about expanding the options in Kubernetes repository](https://github.com/search?q=repo%3Akubernetes%2Fkubernetes%20%22kubectl%20-n%22&type=code)- why do we care about that in Makefile for internal pubsub use?
Also, why we should use longopts for kubectl, but not for `tar`?

---

I believe we are spending too much time on such details